test/test_fsdp.py

```diff
@@ -160,19 +180,24 @@ def run(mode: str, is_fp8: bool):
         model.load_state_dict(torch.load(sd_in_fname))
         optimizer = torch.optim.SGD(model.parameters(), lr=lr)
 
-        for _ in range(N_ITER):
+        def forward_backward():
             optimizer.zero_grad()
             y = model(ref_input)
             y.sum().backward()
             sync_float8_amax_and_scale_history(model)
             optimizer.step()
+            return y
+
+        for _ in range(N_ITER):
+            y = forward_backward()
 
         torch.save(y, output_single_gpu_fname)
         torch.save(model.state_dict(), sd_out_single_gpu_fname)
 
     elif mode == "fsdp":
         WORLD_SIZE = torch.cuda.device_count()
-        args = (is_fp8, emulate, base_dtype)
+        # We only compile for fsdp, and compare the numerics with signle-gpu no-compile
+        args = (is_fp8, emulate, base_dtype, compile_fsdp, fullgraph)
```

Comment:
@vkuzo `compile_fsdp` is used here.

Reply:
yeah I saw this, wasn't sure how it was actually used in code?  I thought this just plumbs it to `args`, and `args` doesn't do anything with it.

Reply:
ooh sorry for the confusion. After "compile_fsdp" is passed to args, it's then passed to fsdp_main and used there
```
def fsdp_main(rank, world_size, args):
    ... ...
    is_fp8, emulate, base_dtype, compile, fullgraph = args
```